scrapers/ManyVids.yml

```diff
@@ -19,4 +29,69 @@ xPathScrapers:
         Name: $details//h4/a/text()
       Image: //div[@id="rmpPlayer"]/@data-video-screenshot
 
-# Last Updated May 15, 2020
+  performerSearch:
+    performer:
+      Name: //*[contains(concat(' ', normalize-space(@class), ' '), ' profile-pic-name ')]/a/text()
+      URL:
+        selector: //*[contains(concat(' ', normalize-space(@class), ' '), ' profile-pic-name ')]/a/@href
+        replace:
+          - regex: '/(?:Store/Videos/)?$'
+            with: /About/
+
+  performerScraper:
+    common:
+      $aboutlabel: //*[@class='mv-about__container__details__list-unit']/span[@class='mv-about__container__details__list-label']
+    performer:
+      Name: //*[@class='mv-about__banner-name ']/text()
+      Gender:
+        selector: $aboutlabel[contains(text(), 'Gender')]/following-sibling::*
+        replace:
+          - regex: Trans
+            with: transgender_female
+      URL: //a[@data-tab='about']/@href
+      Twitter: //*[@class='mv-about__banner-social']/a/i[@class='icon twitter-icon']/../@href
+      Instagram: //*[@class='mv-about__banner-social']/a/i[@class='icon instagram-icon']/../@href
+      Birthdate:
+        selector: $aboutlabel[contains(text(), 'Age')]/following-sibling::*
+        replace:
+          - regex: ' \(.*\)$'
+            with: ''
+        parseDate: 2006.01.02
+      Ethnicity:
+        selector: $aboutlabel[contains(text(), 'Ethnicity')]/following-sibling::*
```

Comment:
i think we need some `replace` in ethnicity to keep the db consistent 
eg `Black / Ebony` -> `black` , `White / Caucasian` -> `white` , `Latino / Hispanic` -> `hispanic`
NewFreeones ,  Babepedia scrapers have most of the cases i think

Reply:
Agreed on your examples, but what about the other ethnicities on MV, such as Native American, Middle Eastern, Alaskan and so on?

Reply:
The first scraper used in stash had only four categories asian, white, black and hispanic to correlate with the asian, caucasian, black and latin from freeones. I think freeones reclassifies the rest into those four categories.
It would make sense to adopt some more ethnicities like stashdb has: `indian, middle eastern, mixed , other` to have at least some distinct values we can switch from later if needed.
That leaves out (assuming `East Indian` corresponds to `indian`) `Pacific Islander` `Alaskan`  and `Native American` from the MV site. Idk for them add them as they are and leave it to the user to edit, add to other ?

Reply:
I opted to keep the extra ones (including East Indian), but lower-cased for at least a bit of consistency